local/install_deps_linux.bash

```diff
@@ -183,8 +184,8 @@ fi
 if [ ! $only_reproduce ]; then
   # Install other dependencies (e.g. bower).
   nodeenv -p --prebuilt
-  npm install -g bower polymer-bundler
-  bower install
+  npm install --unsafe-perm -g bower polymer-bundler
+  bower --allow-root install
```

Comment:
These two flags allowed to support installation in root mode, they are not generally recommended. Can be removed too if we make it explicit to not run install deps as a root user.

Reply:
Add a comment like "# Allow bower and polymer-bundler install for root users as well."

Reply:
Sure!